internal/injector/aspect/context/context.go

```diff
@@ -163,6 +184,8 @@ func (c *context) Child(node dst.Node, property string, index int) AdviceContext
 		sourceParser: c.sourceParser,
 		importPath:   c.importPath,
 		testMain:     c.testMain,
+		typeInfo:     c.typeInfo,
+		nodeMap:      c.nodeMap,
 	}
```

Comment:
The child context should probably also inherit the log value? (Or get its own via a `context.Context` to be provided?)

Reply:
Good catch. I'll pass down the log.